tests/cstor/gtest/test_uzfs.cc

```diff
@@ -1413,6 +1412,7 @@ TEST(uZFSRebuildStart, TestStartRebuild) {
 	EXPECT_EQ(ZVOL_OP_STATUS_FAILED, ((zvol_io_hdr_t *)conn->conn_buf)->status);
 	EXPECT_EQ(2, zinfo->refcnt);
 
+	uzfs_zinfo_set_status(zinfo, ZVOL_STATUS_DEGRADED);
```

Comment:
this looks unnecessary?

Reply:
https://github.com/openebs/cstor/blob/55b3edca2b173b1d444903772bc5ff214779a581/tests/cstor/gtest/test_uzfs.cc#L1225
We are setting it to Healthy in the `SnapCreateSuccess` test case So I set back to `ZVOL_STATUS_DEGRADED` due to the latest changes in the code.